app/services/bulk_upload/sales/year2023/row_parser.rb

```diff
@@ -509,7 +509,13 @@ def valid?
       fields = field_mapping_for_errors[error.attribute] || []
 
       fields.each do |field|
-        unless errors.include?(field)
+        next if errors.include?(field)
+
+        question = log.form.get_question(error.attribute, log)
+
+        if setup_question?(question)
```

Comment:
any reason we're not checking for question.present? here like we do in lettings?

Reply:
Ah, yeah, should probably do that, thanks!